WheelOfFortune/Prompt.cs

```diff
@@ -98,6 +98,15 @@ public static char GetActionType()
         /// </summary>
         public static dynamic PromptAction(Action action)
         {
+            if (action.ActionTypeProperty == Action.ActionType.GuessLetterAction)
+            {
+                Console.WriteLine("Start Guessing: Enter a letter:");
+            }
+            else if(action.ActionTypeProperty == Action.ActionType.SolvePuzzleAction)
+            {
+                Console.WriteLine("What is your solution to the puzzle?");
+            }
+            
```

Comment:
Here, we should also have logic to validate and read the inputs from the user

Reply:
yes I have it locally and will push in different PR..the Prompt.PromptAction will be called fromStartTuen in a coming PR as well